python/cugraph/cugraph/centrality/betweenness_centrality.py

```diff
@@ -58,6 +58,12 @@ def betweenness_centrality(
         Algorithm (2001) to compute exact or approximate betweenness.
         If weights are provided in the edgelist, they will not be used.
 
+        .. deprecated:: 24.08
+
+           Accepting ``networkx.Graph`` is deprecated and will be removed in a
+           future version.  For ``networkx.Graph`` use netwotkx directly with
```

Comment:
typo networkx

Reply:
aggg that will then be in every instance - thanks.  I'll update that